assignment5/Vertex.java

```diff
@@ -0,0 +1,48 @@
+package com.katermar.assignment5;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Created by USER on 7/8/2017.
+ */
+public class Vertex {

```

Comment:
Vertex does not have to be public as it is not necessary to expose it to any other code that uses AlphabetDiscovering class.